src/ape/managers/config.py

```diff
@@ -106,22 +104,6 @@ def extract_config(cls, manifest: PackageManifest, **overrides) -> ApeConfig:
         """
         return ApeConfig.from_manifest(manifest, **overrides)
 
-    def get_config(self, plugin_name: str) -> PluginConfig:
```

Comment:
bug was fixed by deleting this method.
`get_config` still exists on `ApeConfig` class, and thus is still available here, only without swallowing every single error